docs/linux/fs.md

```diff
@@ -491,6 +491,64 @@ setup necessary.
 # mount /dev/sdb /mnt
 ```
 
+### Support for ZNS SSDs.
```

Comment:
No "." at the end of the section title please.
Also add a blank line after the section title.

Reply:
Also, s/ZNS/NVMe ZNS